contents.json

```diff
@@ -12899,6 +12899,10 @@
       "category-ids": [
         "timer"
       ],
+      "screenshots": [
+        "https://i.imgur.com/oHKTHVJ.png",
```

Comment:
The links should be directly to the image, not a webpage
Can you fix both

Reply:
@dkhamsing  Hi! Probably, I misunderstand you, but the both urls are direct link. Can you explain me some more?

Reply:
my mistake!! sorry and thank you